content/monad-transformers.md

```diff
@@ -23,13 +23,96 @@ A `Reader r a` in which the resulting `a` is wrapped in any other monad, i.e. `r
 
 ### StateT
 
-A `State s a` in which the return value `a` and state `s` are wrapped in any other monad, i.e. `s -> m (a, s)`
+A `State s a` in which the return value and state, namely `(a, s)`, are wrapped in any other monad, i.e. `s -> m (a, s)`
 
-### EitherT
```

Comment:
I'm definitely in agreement here. I'm not saying "no subjective content" (since as you say I already wrote some!), but:

1. Objective content is better when possible
2. Subjective content should be called out as subjective/debatable